setup.py

```diff
@@ -106,6 +106,10 @@ def find_version(*file_paths):
     INSTALL_REQUIRES.extend(EXTRAS_REQUIRE["tf"])
     sys.argv.remove("--tf")
 
+if "--all" in sys.argv:
```

Comment:
@FyodorK @vshampor Do you really need it? I mean we can use tf and torch options at the same time.

Reply:
1. this option is needed for bdist installation
2. In case p.1 is questionable -> why this option has implemented in EXTRAS_REQUIRE dict but still not used at all?